src/source/Signaling/LwsApiCalls.c

```diff
@@ -50,7 +50,7 @@ INT32 lwsHttpCallbackRoutine(struct lws* wsi, enum lws_callback_reasons reason,
     customData = lws_get_opaque_user_data(wsi);
     pLwsCallInfo = (PLwsCallInfo) customData;
 
-    lws_set_log_level(LLL_NOTICE | LLL_INFO | LLL_WARN | LLL_ERR, NULL);
+    lws_set_log_level(LLL_NOTICE | LLL_WARN | LLL_ERR, NULL);
```

Comment:
Info level logging is extremely noisy so I've removed it.  With this change the log level is actually the default log level but I'm still calling the API to be explicit so someone doesn't have to search the documentation to figure out what log levels are specified:

https://github.com/warmcat/libwebsockets/blob/82c21ac80c92fcf7f0cc2b7ff50f3a6e2cfe46e4/include/libwebsockets/lws-logs.h#L244

NOTICE level logging is not very noisy and provides useful http call latencies.